public/app/features/inspector/InspectBugReportTab.tsx

```diff
@@ -0,0 +1,125 @@
+  panel: PanelModel;
+}
+
+export function InspectBugReportTab({ dashboard, panel }: Props): JSX.Element {
+  const [sanitizers, setSanitizers] = useState<Sanitizer[]>([]);
+  const { loading, value } = useAsync<string | undefined>(async () => {
+    const items = await new InspectCollector().collect({
+      dashboard,
+      panel,
+      workers: getCollectorWorkers(),
+      sanitizers,
+      type: CollectorType.Panel,
+    });
+
+    return inspectPackager().package(items);
+  }, [dashboard, panel, sanitizers]);
+  const styles = useStyles2(getStyles);
+
+  const onDownloadClick = () => {
+    if (value) {
+      inspectDownloader().startDownload(value);
+    }
+  };
+
+  const onClipboardSuccess = () => {
+    appEvents.emit(AppEvents.alertSuccess, ['Collected data copied to clipboard']);
+  };
+
+  if (loading || !Boolean(value)) {
+    return <LoadingIndicator />;
+  }
+
+  return (
+    <VerticalGroup>
+      <Alert title="Bug report" severity="info">
+        <p>Although we try to minimize errors in Grafana, they can appear in our product from time to time.</p>
+        <p>
+          To help us fix an error, go to
+          <a href="https://github.com/grafana/grafana/issues/new/choose" target="_blank" rel="noopener noreferrer">
+            &nbsp;creating an issue on GitHub&nbsp;
+          </a>
+          and then, add detailed information from here and reproducible steps to help us investigate and fix the error.
+        </p>
+        <p>
+          This feature collects relevant information about your <em>OS</em>, <em>Browser</em>, <em>Grafana</em>,&nbsp;
+          <em>Dashboard</em> & <em>Panel</em> and package them so that it makes it easier for you to attach the
+          essential information when reporting an issue. The collected data is only available on this page, it is not
+          shared or stored in any way.
+        </p>
+        <p>
+          <em>Make sure to scrub sensitive data before you attach it to any issue, otherwise it will become public.</em>
+        </p>
+      </Alert>
+      <div className={styles.sanitizeContainer}>
+        <Field
+          label="Scrub sensitive data"
+          description={
+            <span>
+              Replaces sensitive information in the collected data with <em>******</em>
+            </span>
+          }
+        >
+          <Switch
+            value={Boolean(sanitizers.length)}
+            onChange={(event) => {
+              if (event.currentTarget.checked) {
+                setSanitizers(getCollectorSanitizers());
+              } else {
+                setSanitizers([]);
+              }
+            }}
+          />
+        </Field>
+        <div className={styles.featureBadge}>
+          <FeatureBadge featureState={FeatureState.alpha} />
+        </div>
+      </div>
+      <HorizontalGroup>
+        <Button type="button" icon="download-alt" onClick={onDownloadClick} variant="secondary">
+          Download collected data
+        </Button>
```

Comment:
I was surprised when this downloaded an md file. I was expecting a zip file with dashboard.json, panel.json etc files. 

Do you think we need this button at all? I can imagine the workflow here to be:
1. I want to report an issue.
2. I go to inspect and copy the md.
3. I paste it into the issue.

Hard for me to think about a case when I would download the fie for ltr use.

Reply:
Wow, that is an excellent idea!! ❤️